src/test/java/module18/tests/CrudTestsPetStorePetEndpoint.java

```diff
@@ -16,16 +17,16 @@
 
 public class CrudTestsPetStorePetEndpoint {
 
-    public static final PetStorePetEndpoint PET_STORE_PET_ENDPOINT = new PetStorePetEndpoint();
+    public static final PetStorePetEndpoint instance = new PetStorePetEndpoint();
```

Comment:
private final PetStorePetEndpoint instance = PetStorePetEndpoint().getInstance();
If you use Singleton `PetStorePetEndpoint` should have private constructor.